stub-idp/src/main/java/uk/gov/ida/stub/idp/resources/idp/SingleIdpPromptPageResource.java

```diff
@@ -32,26 +39,39 @@
     private final IdpStubsRepository idpStubsRepository;
     private final ServiceListService serviceListService;
     private final SingleIdpConfiguration singleIdpConfiguration;
+    private final AllIdpsUserRepository allIdpsUserRepository;
```

Comment:
Could probably remove this ... never used.

Reply:
Done.